src/kibana/components/agg_types/controls/values.html

```diff
@@ -0,0 +1,29 @@
+<div class="form-group" ng-controller="agg.type.params.byName.values.controller">
+  <label>Values</label>
+  <div
+    ng-repeat="value in agg.params.values track by $index"
+    class="form-group vis-editor-agg-form-row vis-editor-agg-form-row">
+
+    <input
+      ng-model="agg.params.values[$index]"
+      values-list="agg.params.values"
+      input-focus
+      class="form-control">
+
+    <button type="button" ng-click="remove($index, 1)" class="btn btn-danger btn-xs">
+      <i class="fa fa-times"></i>
+    </button>
+  </div>
+
+  <input ng-model="validLength" name="validLength" required type="hidden">
+  <p ng-show="aggForm.validLength.$invalid" class="text-danger text-center">
+    You must specify at least one value
+  </p>
+
+  <button
+    ng-click="add()"
+    type="button"
+    class="sidebar-item-button primary">
+    <i class="fa fa-plus"></i> Add value
+  </button>
+</div>
```

Comment:
The diff between this file and `src/kibana/components/agg_types/controls/percents.html` is rather small, would you mind trying to consolidate the two? Perhaps 

```diff
1,2c1,2
< <div class="form-group" ng-controller="agg.type.params.byName.percents.controller">
<   <label>Percentiles</label>
---
> <div class="form-group" ng-controller="agg.type.params.byName.values.controller">
>   <label>Values</label>
4c4
<     ng-repeat="value in agg.params.percents track by $index"
---
>     ng-repeat="value in agg.params.values track by $index"
8,9c8,9
<       ng-model="agg.params.percents[$index]"
<       values-list="agg.params.percents"
---
>       ng-model="agg.params.values[$index]"
>       values-list="agg.params.values"
20c20
<     You mush specify at least one percentile
---
>     You must specify at least one value
27c27
<     <i class="fa fa-plus"></i> Add Percent
---
>     <i class="fa fa-plus"></i> Add value
```